client/lib/wpcom-undocumented/lib/site.js

```diff
@@ -40,10 +41,22 @@ var list = function( resourceOptions ) {
 			return '/';
 		} );
 
+		if ( typeof query === 'function' ) {
+			fn = query;
+			query = {};
+		}
+
 		query.apiVersion = resourceOptions.apiVersion;
 
 		path = '/sites/' + this._id + '/' + subpath;
 
+		// wp-api resources
+		if ( '2' === query.apiVersion ) {
+			path = '/sites/' + this._id + '/wp/v2/' + subpath;
+			query.apiNamespace = 'wp';
+			query.locale = i18n.getLocaleSlug();
```

Comment:
Does `locale` do anything? Requesting with `locale=fr` query parameter appears to have no effect.